feat: docs for api migration (#2436)
langchain-ai/docs: src/langsmith/endpoint-deprecation.mdx

---
title: API versioning and support policy
sidebarTitle: Versioning and deprecation
description: Learn about LangSmith API versioning, support policy, and how to upgrade endpoints.
---

> Learn about LangSmith API versioning and support policy for endpoints. For step-by-step upgrade instructions, see the [migration guide](/langsmith/endpoint-migration) for the relevant endpoint.

Reviewer: ```suggestion
This page describes LangSmith API versioning and support policy for endpoints. For step-by-step upgrade instructions, refer to the [migration guide](/langsmith/endpoint-migration) for the relevant endpoint.
```
